static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -40,6 +50,51 @@ type Props = {
   registersMeta?: Record<any, any>;
 };
 
+export function getCoverageColors(
+  organization: Organization,
+  missingData: boolean,
+  lines: [number, string][],
+  lineCoverage?: LineCoverage[]
+): string[] {
+  const shouldShowCodecovData =
+    organization?.features.includes('codecov-stacktrace-integration') &&
+    organization?.codecovAccess;
+
+  if (!shouldShowCodecovData || !missingData || !lineCoverage) {
+    return lines.map(() => 'transparent');
+  }
+
+  let coverageIndex = 0;
+  for (const [index, lc] of lineCoverage.entries()) {
+    if (lc.lineNo === lines[0][0]) {
+      coverageIndex = index;
+      break;
+    }
+  }
+
+  return lines.map(line => {
+    const coverageLine = lineCoverage[coverageIndex];
+    let coverage = Coverage.NOT_APPLICABLE;
+    if (coverageLine.lineNo === line[0]) {
```

Comment:
we don't have strict index check turned on yet, but its worth assuming that it could be undefined
```suggestion
    if (coverageLine?.lineNo === line[0]) {
```